package.json

```diff
@@ -4,7 +4,13 @@
   "description": "Custom CMS Platform",
   "main": "gulpfile.js",
   "dependencies": {
-    "buble": "Noojuno/buble",
+    "buble": "^0.17.0",
+    "clean-css-cli": "*",
+    "es6": "*",
+    "es6-promise": "*",
+    "eslint": "*",
+    "less": "*",
```

Comment:
Looks like some of the dependency's have duplicated. Npm is weird

Reply:
Oh weird

Reply:
yeah, please fix this.